scalafix-tests/unit/src/test/scala-2/scalafix/tests/util/compat/CompatSemanticdb.scala

```diff
@@ -0,0 +1,18 @@
+package scalafix.tests.util.compat
+
+object CompatSemanticdb {
+
+    def scalacOptions(opts: Array[String], src: java.nio.file.Path, target: java.nio.file.Path) : Array[String] = {
```

Comment:
nitpicking, but I would not take the previous opts as parameter, this can be done on the callsite

Reply:
true! I'm such a dummy haha